docs/api/actions-and-connectors/create.asciidoc

```diff
@@ -6,57 +6,95 @@
 
 Creates a connector.
 
-[discrete]
 [[create-connector-api-request]]
 === {api-request-title}
 
 `POST <kibana host>:<port>/api/actions/connector`
 
 `POST <kibana host>:<port>/s/<space_id>/api/actions/connector`
 
-[discrete]
 === {api-prereq-title}
 
 You must have `all` privileges for the *Actions and Connectors* feature in the
 *Management* section of the
 <<kibana-feature-privileges,{kib} feature privileges>>.
 
-[discrete]
 [[create-connector-api-path-params]]
 === {api-path-parms-title}
 
 `space_id`::
   (Optional, string) An identifier for the space. If `space_id` is not provided
   in the URL, the default space is used.
 
-[discrete]
+[role="child_attributes"]
 [[create-connector-api-request-body]]
 === {api-request-body-title}
 
-`name`::
-  (Required, string) The display name for the connector.
+`config`::
+(Required, object) The configuration for the connector. Configuration properties
+vary depending on the connector type. For example:
++
+.Index connectors
```

Comment:
Do you think it'd be useful to link directly to the connector type page here as well?

Reply:
I've added links to the index and Jira connector pages via 0edeac575c53072c44fe09843a997fdcaeed008e. Is this what you had in mind?

Reply:
Yea! Thank you